lms/djangoapps/teams/views.py

```diff
@@ -1164,12 +1164,11 @@ def get(self, request, topic_id, course_id):
         )
         return Response(serializer.data)
 
-
 class MembershipListView(ExpandableFieldViewMixin, GenericAPIView):
     """
         **Use Cases**
 
-            List course team memberships or add a user to a course team.
```

Comment:
I disagree with this change. CourseTeamMembership and CourseTeam are specific django model names.

Reply:
Updated the docstring.